OpenRA.Mods.Common/AI/HackyAI.cs

```diff
@@ -56,6 +56,8 @@ public class BuildingCategories
 			public readonly HashSet<string> Production = new HashSet<string>();
 			public readonly HashSet<string> NavalProduction = new HashSet<string>();
 			public readonly HashSet<string> Silo = new HashSet<string>();
+			public readonly HashSet<string> Radar = new HashSet<string>();
```

Comment:
Buildings like the Gap Gen or RA2's T3 support buildings (ore Purifier, Industrial Plant, Cloning Vats) would also fit to that list, while not fitting to either of this.